pkg/apis/scheduling/register.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package podgroup
+package scheduling
```

Comment:
Physically, I prefer to keep podgroup and elasticquota in different packages, so that they can evolve with different versions over time.

In terms of the API group, I _think_ it's ok to wire both to the same API group (scheduling.sigs.k8s.io), right?
(If not, or not conventional, we can use different sub-groups)

Reply:
No, I tried yesterday. We can't wire the same api group in different dir.

Reply:
Oh, I overlooked one thing: is it possible to keep multiple **kinds** in the same **group**, and let them evolve with different versions? i.e., it's ok to have the latest version of ElasticQuota as v1alpha1 and PodGroup as v1beta1? If so, I think it's good to just use one API group: "scheduling.sigs.k8s.io".